cirq/contrib/acquaintance/executor.py

```diff
@@ -40,14 +40,16 @@ class ExecutionStrategy(metaclass=abc.ABCMeta):
 
     keep_acquaintance = False
 
-    @abc.abstractproperty
+    @abc.abstractmethod
+    @property
```

Comment:
Unfortunately the order matters.  https://docs.python.org/3/library/abc.html#abc.abstractmethod

Reply:
doh